js/entry-points/tracing.js

```diff
@@ -140,6 +140,17 @@ audion.entryPoints.enumAudioBufferProperties_ =
 
 
 /**
+ * A reference to a native function that performs the logic of
+ * Function.prototype.bind([CONSTRUCTOR], arguments ...). We need this reference
+ * because we rely on using that logic to construct various objects, but some
+ * libraries such as prototype.js override the native bind and apply methods.
+ * @private {!Function}
+ */
+audion.entryPoints.nativeBindApplyMethod_ = Function.prototype.apply.bind(
```

Comment:
True. `audion.entryPoints` is a heavily over-used namespace that contains a hodgepodge of logic across several files. This file should use a specific namespace like `audion.entryPoints.tracing`